OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -95,8 +99,9 @@ public List<CPos> FindUnitPathToRange(CPos source, SubCell srcSub, WPos target,
 				cached = true;
 			}
 
-			var mi = self.Info.TraitInfo<MobileInfo>();
-			var li = mi.LocomotorInfo;
+			var mobile = self.Trait<Mobile>();
+			var li = mobile.Info.LocomotorInfo;
+			var mi = mobile.Info;
```

Comment:
If you switch order if these two, you can populate `li` through `mi.LocomotorInfo`.